documentation/docs/liquidity-mining/upcoming/Zilliqa.md

```diff
@@ -1,43 +1,17 @@
 
-## Timeline
-
-This liquidity mining program will run for 12 weeks starting in early February 2020.
-
-## Rewards
-
-Liquidity miners can earn rewards based on a formula that takes into account duration and consistency of orders placed over time, spreads, and sizes. You'll be able to check your earned rewards by minute, and your total earned balance in the "activity" section. 
-
-## How to Participate
-
-1. Register for the program on the liquidity mining web APP by entering your wallet information and read-only exchange API keys. 
-2. Start market making for eligible ZIL trading pairs (excluding ZIL/BTC) and Earn rewards! 😎 (The rewards will be automatically calculated based on your trading acitivities for eligible pairs collected through the exchange APIs you provide.)
-
-!!! note
-    If you want to participate using Hummingbot, you can [download Hummingbot](https://hummingbot.io/download) on the website, or follow the [Quickstart guide](/quickstart/) to install and configure Hummingbot. Configure and run a bot that uses either the [pure market making](/strategies/pure-market-making) or the [cross-exchange market making](/strategies/cross-exchange-market-making) strategy.
-
-## Updates and Announcements
-
-We send out updates and announcements through our [Discord channel](https://discord.hummingbot.io), [Twitter](https://twitter.com/hummingbot_io), and [bi-weekly newsletters](http://hummingbot.substack.com). 
-
-You can also read through the articles in our [blog](https://hummingbot.io/blog/tag/liquidity-mining#tags) where you can find useful information about liquidity mining such as statistics, AMA, interviews from our top liquidity miners, previous month recap, and more.
-
-## Performance
-
-Check your performance in the Activity section within our liquidity mining app. 
-
-## Evaluation
-
-Once the program officially kicks off, the rewards will be paid out **weekly** in USDC based on the **actual trading activity** of the participants. 
-
-All rewards will be subject to the sole discretion of the token issuer and/or the Hummingbot team.
-
-## More Resources
-
-- [Liquidity Mining FAQs](/liquidity-mining/faq/)
-
----
-## Important Disclaimer
-
-Participating users must agree to the [Liquidity Mining Program Terms and Conditions](https://hummingbot.io/liquidity-mining-policy/).
+[Whitepaper](https://docs.zilliqa.com/whitepaper.pdf) | [Twitter](https://twitter.com/zilliqa) | [Telegram](https://t.me/zilliqachat) | [Github](https://github.com/Zilliqa) | [Explorer](https://viewblock.io/zilliqa)
+
+## Eligable pairs and exchanges
+Pair | Exchange 
+---|--- 
+ ZIL/USDT | Binance
+ ZIL/ETH | Binance
+ ZIL/BNB | Binance
+ ZIL/USDT | Huobi Global
+ ZIL/ETH | Huobi Global
+ ZIL/ETH | KuCoin (coming soon)
+
```

Comment:
To make is clear and to avoid error users, we may want to explicitly say that:

"Note: ZIL/BTC is excluded from the campaign".